src/preferences/dialog/dlgprefeq.cpp

```diff
@@ -261,10 +261,10 @@ void DlgPrefEQ::slotPopulateDeckEffectSelectors() {
         }
         //: Displayed when no effect is selected
         box->addItem(tr("None"), QVariant(QString("")));
-        if (selectedEffectId.isNull()) {
+        if (selectedEffectId.isNull() || QString::compare(selectedEffectName, tr("None")) == 0) {
             currentIndex = availableQuickEffects.size(); // selects "None"
         }
-        if (currentIndex < 0 && !selectedEffectName.isEmpty()) {
+        else if (currentIndex < 0 && !selectedEffectName.isEmpty()) {
```

Comment:
Please follow our coding guidelines by placing the else on the same line

Reply:
Just didn't see them beforehand. I will keep it in mind and re-commit.